test/persist/mzcompose.py

```diff
@@ -18,18 +18,35 @@
 ]
 
 
-def workflow_default(c: Composition) -> None:
-    """Run the nemesis for 5 seconds as a smoke test."""
+def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
+    """Run maelstrom against persist"""
+
+    # Please see `docker run materialize/maelstrom-persist:mzbuild-... --help` for
+    # the meaning of the various arguments
+    parser.add_argument(
+        "--time-limit",
+        type=int,
+        default=60,
+    )
+
+    parser.add_argument("--node-count", type=int, default=1)
+    parser.add_argument("--concurrency", type=int, default=2)
+    parser.add_argument("--rate", type=int, default=100)
+    parser.add_argument("--max-txn-length", type=int, default=4)
+
+    args = parser.parse_args()
+
     c.run(
         "maelstrom-persist",
-        "--time-limit=5",
-        "--node-count=1",
-        "--concurrency=2",
-        "--rate=100",
+        f"--time-limit={args.time_limit}",
+        f"--node-count={args.node_count}",
+        f"--concurrency={args.concurrency}",
+        f"--rate={args.rate}",
         "--",
         "maelstrom",
         "--blob-uri=mem://",
         "--consensus-uri=mem://",
```

Comment:
let's toss in `--unreliability` here too (it's a flag for the persistcli invocation, not maelstrom) and set it to `0.1` for the nightly